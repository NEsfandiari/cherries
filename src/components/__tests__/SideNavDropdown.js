import React from 'react'
import renderer from 'react-test-renderer'
import SideNavDropdown from '../atoms/SideNavDropdown'

describe('Loading', () => {
  it('renders correctly', () => {
    const dropDown = [
      {
        name: 'Basics',
        route: 'basics',
      },
      {
        name: 'Minimalist',
        route: 'minimalist',
      },
      {
        name: 'Girlboss',
        route: 'girlboss',
      },
      {
        name: 'Formal',
        route: 'formal',
      },
      {
        name: 'Boho',
        route: 'boho',
      },
      {
        name: 'Fun',
        route: 'fun',
      },
    ]
    const tree = renderer
      .create(<SideNavDropdown dropDown={dropDown} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
