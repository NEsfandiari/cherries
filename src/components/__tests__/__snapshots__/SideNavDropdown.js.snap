// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<li>
  <a
    className="sc-bdVaJa eWijjc"
    onClick={[Function]}
    style={
      Object {
        "color": "#e20031",
        "fontWeight": "bold",
        "textDecoration": "none",
      }
    }
  >
     
    +
  </a>
  <ul
    className="dropdownContent"
    style={
      Object {
        "display": "none",
      }
    }
  >
    <li
      className="dropDownLink"
    >
      <a
        className="sc-bdVaJa eWijjc"
        href="basics"
        style={
          Object {
            "color": "#47525E",
            "fontWeight": "bold",
            "marginLeft": "25px",
            "textDecoration": "none",
          }
        }
      >
        Basics
      </a>
    </li>
    <li
      className="dropDownLink"
    >
      <a
        className="sc-bdVaJa eWijjc"
        href="minimalist"
        style={
          Object {
            "color": "#47525E",
            "fontWeight": "bold",
            "marginLeft": "25px",
            "textDecoration": "none",
          }
        }
      >
        Minimalist
      </a>
    </li>
    <li
      className="dropDownLink"
    >
      <a
        className="sc-bdVaJa eWijjc"
        href="girlboss"
        style={
          Object {
            "color": "#47525E",
            "fontWeight": "bold",
            "marginLeft": "25px",
            "textDecoration": "none",
          }
        }
      >
        Girlboss
      </a>
    </li>
    <li
      className="dropDownLink"
    >
      <a
        className="sc-bdVaJa eWijjc"
        href="formal"
        style={
          Object {
            "color": "#47525E",
            "fontWeight": "bold",
            "marginLeft": "25px",
            "textDecoration": "none",
          }
        }
      >
        Formal
      </a>
    </li>
    <li
      className="dropDownLink"
    >
      <a
        className="sc-bdVaJa eWijjc"
        href="boho"
        style={
          Object {
            "color": "#47525E",
            "fontWeight": "bold",
            "marginLeft": "25px",
            "textDecoration": "none",
          }
        }
      >
        Boho
      </a>
    </li>
    <li
      className="dropDownLink"
    >
      <a
        className="sc-bdVaJa eWijjc"
        href="fun"
        style={
          Object {
            "color": "#47525E",
            "fontWeight": "bold",
            "marginLeft": "25px",
            "textDecoration": "none",
          }
        }
      >
        Fun
      </a>
    </li>
  </ul>
</li>
`;
